e2e/suite_test.go

```diff
@@ -160,7 +163,7 @@ func bootstrapVMSS(ctx context.Context, t *testing.T, r *mrand.Rand, opts *scena
 		if err != nil {
 			t.Error("error polling deleting vmss", vmssName, err)
 		}
-		t.Logf("finished deleting vmss %q", vmssName)
+		log.Printf"finished deleting vmss %q", vmssName)
```

Comment:
seems like a syntax error somewhere around here caused the latest run to fail

Reply:
yeah missing parenthesis :'(